Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 ADD requirements.txt /app/requirements.txt
 
 # TODO: Remove once cassandra-driver issue is fixed (ref: https://datastax-oss.atlassian.net/browse/PYTHON-656)
-RUN pip2 install -U Cython==0.24.1
+RUN pip2 install -U Cython==0.24.1 instana
```

Comment:
doesn't that "instana" belong to requirements.txt

Reply:
@mohabusama to the scene. AFAIK we have it in the dockerfile because it's optional and we don't want this dependency to be mandatory.